backend/setup.cfg

```diff
@@ -8,6 +8,8 @@ package_dir =
 
 install_requires =
     fastapi==0.109.0
```

Comment:
A couple of python package items which we can go ahead and add here:
1) After line 7, add something like:

```
    rope.db = db
```

2) Create an empty `./backend/db/__init__.py` file